importer/tests/test_xanalytics.py

```diff
@@ -0,0 +1,89 @@
+"""
+Tests for hitting the xanalytics Celery tasks.
+"""
+
+import json
+import logging
+
+from django.conf import settings
+import responses
+
+from importer.tasks import populate_xanalytics_fields, check_for_results
+from learningresources.tests.base import LoreTestCase
+
+log = logging.getLogger(__name__)
+# pylint: disable=no-self-use
+
+
+class TestAPI(LoreTestCase):
```

Comment:
I think you missed them a little bit, although you did leave code review notes in that file. :o)

https://github.com/mitodl/lore/blob/feature/skm/166_xanalytics/xanalytics/tests/test_api.py